test/cases/testdata/jwtdecodeverify/test-jwtdecodeverify-invalid-nbf-type.yaml

```diff
@@ -0,0 +1,15 @@
+cases:
+  - data:
+    modules:
+      - |
+        package generated
+        
+        token := io.jwt.encode_sign({"alg": "HS256"}, {"nbf": null}, {"kty": "oct", "k": base64url.encode_no_pad("42")})
```

Comment:
Maybe have one of the tests take an invalid value other than null, like a string or an object.

Reply:
Good idea. I changed nbf to pass a string.